build-parent/pom.xml

```diff
@@ -73,12 +73,12 @@
         <volume.access.modifier>:Z</volume.access.modifier>
 
         <!-- Defaults for integration tests -->
-        <elasticsearch-server.version>8.18.2</elasticsearch-server.version>
+        <elasticsearch-server.version>9.0.1</elasticsearch-server.version>
```

Comment:
I see `9.0.1` here and `9.0.2` above?

Reply:
This one is the Elasticsearch server (container) version, the other one is the client. They don't need to be aligned perfectly.

But it's better to be on the latest micro, so I'll align.